repository/src/main/java/ca/bc/gov/repository/services/OrganDonorRepository.kt

```diff
@@ -0,0 +1,35 @@
+package ca.bc.gov.repository.services
+
+import ca.bc.gov.common.const.DATABASE_ERROR
+import ca.bc.gov.common.exceptions.MyHealthException
+import ca.bc.gov.common.model.services.OrganDonationDto
+import ca.bc.gov.data.datasource.local.OrganDonationLocalDataSource
+import ca.bc.gov.data.datasource.remote.OrganDonorRemoteDataSource
+import javax.inject.Inject
+
+/**
+ * @author Pinakin Kansara
+ */
+class OrganDonorRepository @Inject constructor(
+    private val organDonorRemoteDataSource: OrganDonorRemoteDataSource,
+    private val organDonationLocalDataSource: OrganDonationLocalDataSource
```

Comment:
It doesn't need to be in this PR, but I'd suggest standardize the naming and replace all **organDonation** by **organDonor**

Reply:
updated the names